uiowa/uiowa
# Hotfix: GC program type access for non-Admins

# Updates

* Turns off Bit module for default platform, since it wasn't being used for anything else yet.
* Transitions the formerly Bit program type to using Fragments instead.
* And indirectly resolves #2786.

# How to test

* `blt ds --site=grad.uiowa.edu`
* `drush @grad.local admin/people`
* Masquerade as Marie Wakeman
* Visit https://grad.local.drupal.uiowa.edu/admin/content/fragments
* Should not see 'Access denied' page
